api/enki.py

```diff
@@ -749,6 +749,11 @@ def update_circulation(self, since):
                 metadata = self.api.get_item(license_pool.identifier.identifier)
                 if metadata:
                     self.process_book(metadata)
+            else:
+                license_pool, made_changes = circulation.apply(
+                    self._db, self.collection
+                )
```

Comment:
This way it doesn't call circulation.apply twice - if bibliographic data doesn't exist yet, it will get that first.